esmvaltool/diag_scripts/hydrology/wflow.py

```diff
@@ -2,12 +2,12 @@
 import logging
 from pathlib import Path
 
-import iris
 import numpy as np
 from osgeo import gdal
+from esmvalcore.preprocessor import regrid
+import iris
```

Comment:
I am in awe about how Codacy is not complaining about this import order :grin:

Reply:
Me too, but I already asked Sarah to check something like that lately and it turns out I was the one that's wrong... This one is even more strange though!

Reply:
hmmm, the prospector did not complain about it, and the pylint in vscode says:` third party import "from esmvalcore.preprocessor import regrid" should be placed before "import iris"pylint`.

Reply:
it's possible prospector didn't run the pylint bit and the CI let it pass - it does that from time to time when there are lots of commits

Reply:
looks better now :+1: